stories/organisms/Dialogs.stories.tsx

```diff
@@ -93,3 +95,56 @@ export const _Confirm: ConfirmStory = {
         );
     },
 };
+
+export const _Dialog: ConfirmStory = {
+    name: 'Dialog',
+    args: {
+        title: 'You’re about to delete this page',
+        contentLabel: 'content label',
+        children: (
+            <div>
+                Before you delete it permanently, there’s some things you should know:
+                <ul>
+                    <li>4 pages have links to this page that will break </li>
+                    <li>2 child pages will be left behind in the page tree</li>
+                </ul>
+            </div>
+        ),
+    },
+    render: (args) => {
+        const [showAlert, setShowAlert] = React.useState(false);
+
+        const handleButtonClick = () => {
+            setShowAlert(true);
+        };
+
+        const handleAccept = () => {
+            setShowAlert(false);
+            console.log('Dialog has been accepted');
+        };
+
+        const handleCancel = () => {
+            setShowAlert(false);
+            console.log('Dialog has been cancelled');
+        };
+
+        return (
+            <>
+                <Button onClick={handleButtonClick}>Click me to see a confirm dialog</Button>
+                <Dialog
+                    {...args}
+                    isOpen={showAlert}
+                    onClose={handleCancel}
+                    acceptButton={{
+                        onClick: handleAccept,
+                        label: 'Delete',
```

Comment:
Maybe better to change the variant to critical to be consistent with Delete action